client/js/components/user/DpUserList/DpUserFormFields.vue

```diff
@@ -379,7 +379,7 @@ export default {
     },
 
     resetData () {
-      if (hasPermission('area_organisations') === false) {
+      if (hasPermission('feature_organisation_user_list') === false) {
```

Comment:
```suggestion
      if (!hasPermission('feature_organisation_user_list')) {
```

Reply:
done in [41b9b0c](https://github.com/demos-europe/demosplan-core/pull/3475/commits/41b9b0cc616c8556a288e1c5d05e8b4613494e1b)